config/routes.rb

```diff
@@ -14,4 +14,13 @@
   # root 'users#index'
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] Great job working on this milestone. To always keep our codebase professional, we have to remove commented lines of code. Kindly delete lines 14 and 16.

Reply:
resolved